gitgud/skills/user_messages.py

```diff
@@ -74,3 +74,15 @@ def all_levels_complete():
 def default_fail():
     print('Level not complete, keep trying. "git gud reset" to start from scratch.')
 
+@separated
+def handle_solutions_messages(level, message_id):
```

Comment:
I think we need to split this up and give appropriate names. When you're calling it, you have no idea what you're going to get out of it.

A good example might be `no_solution_available`, `confirm_show_solution` or simply `show_solution`.
Whatever the name, it should be useful, but the method should only do one thing